### PR TITLE
EN-722 NotFoundPage component

### PR DESCRIPTION
**Pull request by mcarta-rt**
NotFoundPage component created (EN-723)
CSS created (EN-724)
unit test created (EN-725)
showcase updated (EN-726)

### CODE REVIEW

File: __mocks__/react-intl.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+
+// eslint-disable-next-line
+export const FormattedMessage = () => (
```
Comment (nicpuddu) [inline __mocks__/react-intl.js]: if this is the only export why is the default missing?

File: src/locales/en.js [hunk 2]
```diff
@@ -11,5 +11,8 @@ export default {
     'fcc.loginPage.title': 'the dxp platform',
     'fcc.loginPage.subtitle': 'for the ux convergence',
     'fcc.loginPage.abstract': 'Entando is the lightest, open source Digital Experience Platform (DXP) for modern applications. Entando harmonizes customer experience across the omnichannel applying the techniques of modern software practices to enterprise applications. Learn quickly, develop easily, deploy rapidly.',
+    'fcc.notfoundPage.title': '404',
+    'fcc.notfoundPage.abstract': 'The request page was not found',
```
Comment (nicpuddu) [inline src/locales/en.js]: `requested` not `request`

File: src/locales/it.js [hunk 3]
```diff
@@ -11,5 +11,8 @@ export default {
     'fcc.loginPage.title': 'the dxp platform',
     'fcc.loginPage.subtitle': 'for the ux convergence',
     'fcc.loginPage.abstract': 'Entando is the lightest, open source Digital Experience Platform (DXP) for modern applications. Entando harmonizes customer experience across the omnichannel applying the techniques of modern software practices to enterprise applications. Learn quickly, develop easily, deploy rapidly.',
+    'fcc.notfoundPage.title': '404',
+    'fcc.notfoundPage.abstract': 'The request page was not found',
```
Comment (nicpuddu) [inline src/locales/it.js]: same applies here

File: __mocks__/react-intl.js (hunk 1 above)
Comment (mcarta-rt) [reply __mocks__/react-intl.js]: `FormattedMessage` is exported by name export from the original package `react-intl`, so the mock should be follow the same logic to ensure the correct operation.

**Pull request merged**